tags/shortcuts.tag.html

```diff
@@ -5,18 +5,20 @@
             $.dialog({
                 title: 'Keymap',
                 content: `
+                <p>File Management</p>
                 <table>
-                    <tr><td><kbd>del</kbd></td><td> Delete selected shapes or feature points.<td></tr>
-                    <tr><td><kbd>Alt</kbd> + <kbd>a</kbd></td><td> Select all shapes.<td></tr>
-                    <tr><td><kbd>Shift</kbd> + <kbd>Alt</kbd> + <kbd>o</kbd></td><td> Open image folder.<td></tr>
                     <tr><td><kbd>Alt</kbd> + <kbd>o</kbd></td><td> Open images.<td></tr>
+                    <tr><td><kbd>Shift</kbd> + <kbd>Alt</kbd> + <kbd>o</kbd></td><td> Open image folder.<td></tr>
                     <tr><td><kbd>Ctrl</kbd> + <kbd>i</kbd></td><td> Import data file.<td></tr>
                     <tr><td><kbd>Ctrl</kbd> + <kbd>e</kbd></td><td> Export data file.<td></tr>
                     <tr><td><kbd>Alt</kbd> + <kbd>Left/Right Arrow</kbd></td><td> Navigate & load images from slider to workarea.<td></tr>
                 </table>
                 <br>
-                <p>Manipulate shape</p>
```

Comment:
It is not image manipulation but shape or rather label shape

Reply:
I changed image to shape.